wgpu-native/src/instance.rs

```diff
@@ -40,14 +43,15 @@ pub fn create_instance() -> ::back::Instance {
     ::back::Instance::create("wgpu", 1)
 }
 
-#[no_mangle]
 #[cfg(feature = "local")]
+#[no_mangle]
 pub extern "C" fn wgpu_create_instance() -> InstanceId {
     let inst = create_instance();
     HUB.instances.register_local(inst)
 }
 
-#[cfg(all(feature = "local", feature = "window-winit"))]
+
+#[cfg(feature = "window-winit")]
 #[no_mangle]
 pub extern "C" fn wgpu_instance_create_surface_from_winit(
```

Comment:
I see it as `winit-window` basically requires `local`. And yes, it shouldn't be in `wgpu.h` at all, ideally. Problem is that `cbindgen` runs a parser that isn't completely aware of the features we provide... it can only gate things based on the defines for features (as far as I understand), so we end up with:
```cpp
#if defined(WGPU_WINDOW_WINIT)
WGPUSurfaceId wgpu_instance_create_surface_from_winit(WGPUInstanceId instance_id,
                                                      const WGPUWindow *window);
#endif
```
This is the best I could do to make it disabled.